pyanaconda/modules/payloads/source/source_base.py

```diff
@@ -84,3 +89,50 @@ def tear_down_with_tasks(self):
         :rtype: list[task]
         """
         pass
+
+
+class MountingSourceBase(PayloadSourceBase, ABC):
+    """Base class for sources that use mounting.
+
+    Implements some common functionality, most notably generation of mount point paths.
+    """
+    # pylint: disable=abstract-method
+
+    counter = 0
+
+    def __init__(self):
+        super().__init__()
+        self._mount_point = self._generate_mount_point()
+
+    def _generate_mount_point(self):
+        result = "/mnt/anaconda-sources/{}-{:0>4}".format(self.type.value.lower(), self.counter)
```

Comment:
Please use the same path we are using now. That means ``/run/install/...``

Reply:
Done: Now paths are like /run/install/source-TYPE-NNNN.